README.md

```diff
@@ -413,13 +413,15 @@ spec:
   ...
   glanceAPIInternal:
     ...
-    externalEndpoints:
-      - endpoint: internal
-        ipAddressPool: osp-internalapi
-        loadBalancerIPs:
-        - "172.17.0.202"
-        sharedIP: true
-        sharedIPKey: ""
+    override:
```

Comment:
since we don't have a sample for the current one, I planned to only have a network-isolation sample in the openstack-operator as it is today. but we can also add a sample here

Reply:
ack, we can provide a sample based on the basic `openstackcontrolplane` in a follow up, the description in the `Readme` is enough.